src/idl_gen_cpp.cpp

```diff
@@ -2826,13 +2826,13 @@ class CppGenerator : public BaseGenerator {
               field.value.type.base_type == BASE_TYPE_STRUCT &&
               !IsStruct(field.value.type);
           if (is_pointer) {
-            code += "if(_o->" + Name(field) + ") { ";
+            code += "{ if(_o->" + Name(field) + ") { ";
```

Comment:
it seems adding these `{}` is the only change in the code generator, yet I see all sorts of other changes in the generated code, how come?

Reply:
The ```{}``` are the second commit to fix the Travis build because it was complaining about the dangling ```else```. The total diff has several lines in it. I forgot to rebase it to squash this into a single commit. 

Would it be better to squash this branch or make a new branch/pull request after its squashed?